src/modules/cmdpal/extensionsdk/Microsoft.CmdPal.Extensions.Helpers/Utilities.cs

```diff
@@ -0,0 +1,83 @@
+﻿// Copyright (c) Microsoft Corporation
+// The Microsoft Corporation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System.Runtime.InteropServices;
+using Windows.Win32;
+using Windows.Win32.Foundation;
+using Windows.Win32.UI.Shell;
+
+namespace Microsoft.CmdPal.Extensions.Helpers;
+
+[System.Diagnostics.CodeAnalysis.SuppressMessage("StyleCop.CSharp.NamingRules", "SA1312:Variable names should begin with lower-case letter", Justification = "It's a Windows constants a ")]
+public class Utilities
+{
+    /// <summary>
+    /// Used to produce a path to a settings folder which your app can use.
+    /// If your app is running packaged, this will return the redirected local
+    /// app data path (Packages/{your_pfn}/LocalState). If not, it'll return
+    /// %LOCALAPPDATA%\{settingsFolderName}.
+    ///
+    /// Does not ensure that the directory exists. Callers should call
+    /// CreateDirectory before writing settings files to this directory.
+    /// </summary>
+    /// <example>
+    /// var directory = Utilities.BaseSettingsPath("Some.Unique.String.Here");
+    /// Directory.CreateDirectory(directory);
+    /// </example>
+    /// <param name="settingsFolderName">A fallback directory name to use
+    /// inside of %LocalAppData%, in the case this app is not currently running
+    /// in a package context</param>
+    /// <returns>The path to a folder to use for storing settings.</returns>
+    public static string BaseSettingsPath(string settingsFolderName)
+    {
+        // KF_FLAG_FORCE_APP_DATA_REDIRECTION, when engaged, causes SHGet... to return
+        // the new AppModel paths (Packages/xxx/RoamingState, etc.) for standard path requests.
+        // Using this flag allows us to avoid Windows.Storage.ApplicationData completely.
+        var FOLDERID_LocalAppData = new Guid("F1B32785-6FBA-4FCF-9D55-7B8E7F157091");
```

Comment:
not a var; a const? can it be a const?

Reply:
![image](https://github.com/user-attachments/assets/0b4cd92c-9031-471e-9c42-6ca12124e746)

:shrug: